controller/space.go

```diff
@@ -209,6 +209,17 @@ func (c *SpaceController) Delete(ctx *app.DeleteSpaceContext) error {
 		// Delete associated space resource
 		resource, err := appl.SpaceResources().LoadBySpace(ctx, &ctx.SpaceID)
 		if err != nil {
+			if notFound, _ := errors.IsNotFoundError(err); notFound {
+				// Space resource is not found.
+				// It can happen to old spaces if Keycloak failed to create a space resource for some reason during space creation.
+				// New spaces won't be created if the space resource creation failed but we have to ignore this errors for old spaces.
+				log.Error(ctx, map[string]interface{}{
```

Comment:
Should it be a 'warning' because we really don't care whether the keycloak resource was deleted as long as the space info is deleted from the DB properly ?

Reply:
It's an error actually. It should not happen and if it did then IMO it's better to log as an error. It's just not a fatal error and we can proceed.
We will handle it better when move to our own authZ solution.